transifex/utils/common.py

```diff
@@ -67,11 +67,45 @@ def repo_context(*args, **kwargs):
         repo.cleanup()
 
 
+# Merge methods supported by the Github API. https://developer.github.com/v3/pulls/#merge-a-pull-request-merge-button
+MERGE_METHODS = {'merge', 'squash', 'rebase'}
+
+
+GithubPullRequest = github.PullRequest.PullRequest
+class ExtendedPullRequest(GithubPullRequest):
+    def merge(self, commit_message=github.GithubObject.NotSet, merge_method=None):
```

Comment:
I would default `merge_method` to `rebase`. As you noted, most of our repos require this, and in those that don't, rebase merging is still allowed (and preferable to creating merge bubbles).